File: week-02/day-01/cuboid.py
# Write a program that stores 3 sides of a cuboid as variables (doubles)
# The program should write the surface area and volume of the cuboid like:
# 
# Surface Area: 600
# Volume: 1000
a = int(input("Kérem az a oldalt: \n"))

print("Surface area: " + str(a*6) + "\n")
print("Volume: " + str(a**3))

File: week-08/day-01/12_cuboid/cuboid.js
// Write a program that stores 3 sides of a cuboid as variables (floats)
// The program should write the surface area and volume of the cuboid like:
//
// Surface Area: 600
// Volume: 1000

var cuboidSideSize = 10;
var surfaceArea = Math.pow(cuboidSideSize,2)*6;
var cuboidVolume = Math.pow(cuboidSideSize,3);

console.log("Surface Area: "+ surfaceArea +"\n"+
            "Volume: " + cuboidVolume);